src/resources/espnow.cc

```diff
@@ -46,43 +49,67 @@ struct DataGram {
 
 const int kInvalidEspNow = -1;
 
+// These constants must be synchronized with the Toit code.
+const int kDataAvailableState = 1 << 0;
+const int kSendDoneState = 1 << 1;
+
+enum class EspNowEvent {
+  new_data_available = 1 << 0,
```

Comment:
`SCREAMING_CASE` and why bother with the bit assignment? That just seems confusing since you can only have on `EspNowEvent` value.

Reply:
Wanted to make it similar to the `kDataAvailableState` and `KSendDoneState` but really not worth it.
done.